Libplanet.Explorer.UnitTests/GraphTypes/BlockTypeTest.cs

```diff
@@ -53,28 +54,32 @@ public async void Query()
 
             ExecutionResult result =
                 await ExecuteQueryAsync<BlockType<NullAction>>(query, source: block);
-            Dictionary<string, object> resultData = (Dictionary<string, object>)result.Data;
+            Dictionary<string, object> resultData =
+                (Dictionary<string, object>)((ExecutionNode) result.Data!)?.ToValue()!;
             Assert.Null(result.Errors);
-            Assert.Equal(block.Index, resultData["index"]);
-            Assert.Equal(
-                ByteUtil.Hex(block.Hash.ToByteArray()),
-                resultData["hash"]);
-            Assert.Equal(block.Difficulty, resultData["difficulty"]);
-            Assert.Equal(
-                block.TotalDifficulty,
-                resultData["totalDifficulty"]);
-            Assert.Equal(
-                block.Miner.ToString(),
-                resultData["miner"]);
-            Assert.Equal(
-                ByteUtil.Hex(block.Nonce.ToByteArray()),
-                resultData["nonce"]);
-            Assert.Equal(
-                new DateTimeOffsetGraphType().Serialize(block.Timestamp),
-                resultData["timestamp"]);
-            Assert.Equal(
-                ByteUtil.Hex(block.StateRootHash.ToByteArray()),
-                resultData["stateRootHash"]);
+            if (resultData != null)
```

Comment:
In what cases `resultData` is not present?

Reply:
I checked again just now and it appears that there isn't a case where the `resultData` is null. I might have included this code before fixing another API. I'll remove this part along with https://github.com/planetarium/libplanet/pull/1690/files#diff-aeb0e1c689f74d1eefa3a1ca8914edc2c4c379114561a8011f9b52e610d5b5a1R45-R46. Thanks for checking. 🙏

Reply:
Note that there is another similar pattern in `TransactionTypeTest.Query()` method as well.